src/example-api/example-api.csproj

```diff
@@ -1,23 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>example-api</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Folder Include="wwwroot\" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.3.19" />
-    <PackageReference Include="AWSSDK.SimpleSystemsManagement" Version="3.3.14" />

```

Comment:
If you want, I could restore the dependencies in the example project. They are not strictly needed since the library includes these references, but maybe you want to showcase that it's possible to override the library dependencies with your own if you want to.